hack/release.toml

```diff
@@ -0,0 +1,47 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "Talos"
+github_repo = "talos-systems/talos"
+match_deps = "^github.com/(talos-systems/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v0.8.0"
+
+pre_release = true
+
+preface = """\
+"""
+
+[notes]
+
+    [notes.features]
+        title = "New Features"
+        description = """\
+* Control Plane as Staitc Pods
+* ECDSA Keys for Kubernetes PKI
+* Disk Encryption
+* Virtual Shared IP for Control Plane Endpoint
```

Comment:
Maybe should also add ipv6 and wireguard support here.

Reply:
I was simply following https://www.talos.dev/docs/v0.9/introduction/what-is-new/, we can update it for the next release if we have more docs, there's nothing to explain what DHCPv6 and Wireguard is

Reply:
ah, I see 👍